src/common/perf_counters.h

```diff
@@ -140,6 +140,11 @@ class PerfCounters
 	  0; // no change, report avg over all time
       return (cur.second - last.second) / (cur.first - last.first);
     }
+    T real_time() const {
+      if (cur.first == last.first)
+        return 0;
+      return (cur.second - last.second) / (cur.first - last.first);
+    }
```

Comment:
These call sites appear to be the only users of the avg() method.. let's just change that?  the 'report avg over all time' doesn't make much sense anyway!